pallets/liquidity-mining/src/lib.rs

```diff
@@ -1182,7 +1189,19 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 							farm_entry.updated_at = current_period;
 
 							let pot = Self::pot_account_id().ok_or(Error::<T, I>::ErrorGetAccountId)?;
-							T::MultiCurrency::transfer(global_farm.reward_currency, &pot, &who, rewards)?;
+							let ed = T::AssetRegistry::existential_deposit(global_farm.reward_currency)
+								.ok_or(Error::<T, I>::NoExistentialDepositForAsset)?;
```

Comment:
This should be `InconsistentState` error - you can't have asset without ED. `T::AssetRegistry::existential_deposit` returns `None` only if asset is not registered and something is very, very wrong if this happen